django/contrib/auth/forms.py

```diff
@@ -104,7 +104,7 @@ class UserCreationForm(forms.ModelForm):
     )
 
     class Meta:
-        model = User
```

Comment:
It was grabbing the default User model, if you specified a custom `AUTH_USER_MODEL` in the settings it will not take effect and will raise `AttributeError`: 
```
Manager isn't available; 'auth.User' has been swapped for 'myauth.User'
```